script.js

```diff
@@ -68,29 +72,33 @@ function displayStories(
     let board = "todo";
     let storyUrl = story.url;
     let statusBadge = story.current_state;
-    let estimate = story.estimate;    
-    appendStory(story, board, storyUrl, statusBadge, estimate);
+    let estimate = story.estimate;
+    let storyId = story.id; 
+    appendStory(story, board, storyUrl, statusBadge, estimate, storyId);
   });
   reviewStories.forEach(function(story) {
     let board = "ready-for-review";
     let storyUrl = story.url;
     let statusBadge = story.current_state;
     let estimate = story.estimate;    
-    appendStory(story, board, storyUrl, statusBadge, estimate);
+    let storyId = story.id; 
+    appendStory(story, board, storyUrl, statusBadge, estimate, storyId);
   });
   inprogressStories.forEach(function(story) {
     let board = "in-progress";
     let storyUrl = story.url;
     let statusBadge = story.current_state;
     let estimate = story.estimate;    
-    appendStory(story, board, storyUrl, statusBadge, estimate);
+    let storyId = story.id; 
+    appendStory(story, board, storyUrl, statusBadge, estimate, storyId);
   });
   doneStories.forEach(function(story) {
     let board = "done";
     let storyUrl = story.url;
     let statusBadge = story.current_state;
     let estimate = story.estimate;    
-    appendStory(story, board, storyUrl, statusBadge, estimate);
+    let storyId = story.id; 
+    appendStory(story, board, storyUrl, statusBadge, estimate, storyId);
```

Comment:
There's a lot of duplication in these setup loops. Is there any way you can DRY it up?